reference/orders.v3.yml

```
@@ -712,7 +712,7 @@ paths:
                         }
                     }
         '503':
-          description: All failure response.
+          description: "All failure response. \nHTTP status code for the entire batch request to roll up to the most severe individual failure to the whole request."
```

```suggestion
          description: "All <<what?>> failed. \nThe HTTP status code for the entire batch request rolls the most severe individual failure up to the whole request."
```
I'm not sure what this means. Rolls up? is this a directional/spatial metaphor for data/error flow?